README.md

```diff
@@ -1,2 +1,201 @@
+# OUTPUT SHOULD BE LIKE BELOW
+#
+# Aug 13 16:30:20 commerciotestnet-node4 cnd[351]: I[2019-08-13|16:30:20.722] Executed block                               module=state height=1 validTxs=0 invalidTxs=0
+# Aug 13 16:30:20 commerciotestnet-node4 cnd[351]: I[2019-08-13|16:30:20.728] Committed state                              module=state height=1 txs=0 appHash=9815044185EB222CE9084AA467A156DFE6B4A0B1BAAC6751DE86BB31C83C4B08
+# Aug 13 16:30:20 commerciotestnet-node4 cnd[351]: I[2019-08-13|16:30:20.745] Executed block                               module=state height=2 validTxs=0 invalidTxs=0
+# Aug 13 16:30:20 commerciotestnet-node4 cnd[351]: I[2019-08-13|16:30:20.751] Committed state                              module=state height=2 txs=0 appHash=96BFD9C8714A79193A7913E5F091470691B195E1E6F028BC46D6B1423F7508A5
+# Aug 13 16:30:20 commerciotestnet-node4 cnd[351]: I[2019-08-13|16:30:20.771] Executed block                               module=state height=3 validTxs=0 invalidTxs=0
+```
+
+## 6. Add wallet key, get tokens and create a validator
+Inside the testnet we don't use the Ledger. 
+However, if you wish to do so, please add the `--ledger` flat to any command.
+    
+**ATTENTION**  
+Please remember to copy the 24 words seed phrase in a secure place.  
+They are your memonic and if you loose them you lose all your tokens and the whole access to your validator.  
+
+```shell
+cncli keys add $NODENAME
+# Enter a password that you can remember
+```
+
+Copy your public address. It should have the format `did:com:<data>`.
+
+```shell
+cncli keys show $NODENAME --address
+```
+    
+From now on we will refer to the value of your public address using the `<your pub addr>` notation.
+
+In order to receive your tokens, please send your `<your pub addr>` inside our 
+[Telegram group](https://t.me/commercionetworkvipsTelegram) requesting them. 
+We will make sure to send them to you as soon as possible.
+
+Once you've been confirmed the successful transaction, please check using the following command:
+```shell
+cncli query account <your pub addr> --chain-id commercio-testnet2000
+# Output should like this
+# ...
+#   - denom: ucommercio
+#    amount: "52000000000"
+# ...
+```
+
+Now you can create validator. If you want you can fill follow parameters
+* `--details`: add a brief description about your node or your company
+* `--identity`: your [Keybase](https://keybase.io) identity
+* `--website`: a public site of your node or your company
+
+
+```shell
+cncli tx staking create-validator \
+  --amount=50000000000ucommercio \
+  --pubkey=$(cnd tendermint show-validator) \
+  --moniker="$NODENAME" \
+  --chain-id=commercio-testnet2000 \
+  --identity="" --website="" --details="" \
+  --commission-rate="0.10" --commission-max-rate="0.20" \
+  --commission-max-change-rate="0.01" --min-self-delegation="1" \
+  --from=<your pub addr> \
+  -y
+# Output should like this
+# ...
+# rawlog: '[{"msg_index":0,"success":true,"log":""}]'
+# ...
+```
+
```

Comment:
## 7. Confirm your validator is running.
Your validator is active if the following command returns anything:
```shell
cncli query staking validators --chain-id $CHAINID | fgrep $(cnd tendermint show-validator)
```
You should now see your validator in [Commercio explorer](https://test.explorer.commercio.network/it/validators)